packages/ia-zendesk-help-widget/test/zendesk-help-widget.test.js

```diff
@@ -0,0 +1,70 @@
+import {
+  html,
+  fixture,
+  expect
+} from '@open-wc/testing';
+import '../src/ia-zendesk-help-widget';
+
+const testwidgetSrc = '/base/test/mock-zen-desk.js';
+
+const component = (properties = {
+  widgetSrc: testwidgetSrc
+}) => (
+  html`
+    <ia-zendesk-help-widget
+      widgetSrc=${properties.widgetSrc}
+    ></ia-zendesk-help-widget>`
+);
+
+let testableVariable = false;
+function testableCode() {
+  let counter = 1;
+  const interval = setInterval(() => {
+    counter += 1;
+
+    if (counter >= 5) {
+      testableVariable = true;
+      clearInterval(interval);
+    }
+  }, 200);
+
+  return interval;
+}
+
+describe('<zendesk-help-widget>', () => {
```

Comment:
Not a blocker, but one spot I think could use unit testing is the UI change when the media query changes.